Doesn't yet support latest pypy

I'm currently limiting pypy to 5.3.1 in .travis.yml

The latest pypy triggers an error like this:

```
Building wheels for collected packages: dockerfile
  Running setup.py bdist_wheel for dockerfile: started
  Running setup.py bdist_wheel for dockerfile: finished with status 'error'
  Complete output from command /home/travis/gopath/src/github.com/asottile/dockerfile/.tox/pypy/bin/pypy -u -c "import setuptools, tokenize;__file__='/tmp/pip-lRJy3v-build/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" bdist_wheel -d /tmp/tmpwvn1yZpip-wheel- --python-tag pp257:
  running bdist_wheel
  running build
  running build_ext
  $ GOPATH=/tmp/tmp7i_pYi go get -d
  /tmp/cctGS3Lw.o: In function `main':
  test.c:(.text+0xa): undefined reference to `f'
  collect2: error: ld returned 1 exit status
  $ CGO_CFLAGS=-I/home/travis/gopath/src/github.com/asottile/dockerfile/.tox/pypy/include CGO_LDFLAGS=-Wl,--unresolved-symbols=ignore-all GOPATH=/tmp/tmp7i_pYi go build -buildmode=c-shared -o /tmp/pip-lRJy3v-build/build/lib.linux-x86_64-2.7/dockerfile.pypy-41.so
  # github.com/asottile/dockerfile/pylib
  ./main.go:56: cannot use C.Py_ssize_t(len(strs)) (type C.Py_ssize_t) as type C.long in argument to _Cfunc_PyTuple_New
  ./main.go:63: cannot use C.Py_ssize_t(i) (type C.Py_ssize_t) as type C.long in argument to _Cfunc_PyTuple_SetItem
  ./main.go:89: cannot use C.Py_ssize_t(len(cmds)) (type C.Py_ssize_t) as type C.long in argument to _Cfunc_PyTuple_New
  ./main.go:122: cannot use C.Py_ssize_t(i) (type C.Py_ssize_t) as type C.long in argument to _Cfunc_PyTuple_SetItem
  Traceback (most recent call last):
    File "<module>", line 1, in <module>
    File "/tmp/pip-lRJy3v-build/setup.py", line 37, in <module>
      setup_requires=['setuptools-golang>=0.2.0'],
    File "/opt/python/pypy-5.7.1/lib-python/2.7/distutils/core.py", line 151, in setup
      dist.run_commands()
    File "/opt/python/pypy-5.7.1/lib-python/2.7/distutils/dist.py", line 953, in run_commands
      self.run_command(cmd)
    File "/opt/python/pypy-5.7.1/lib-python/2.7/distutils/dist.py", line 972, in run_command
      cmd_obj.run()
    File "/home/travis/gopath/src/github.com/asottile/dockerfile/.tox/pypy/site-packages/wheel/bdist_wheel.py", line 179, in run
      self.run_command('build')
    File "/opt/python/pypy-5.7.1/lib-python/2.7/distutils/cmd.py", line 334, in run_command
      self.distribution.run_command(command)
    File "/opt/python/pypy-5.7.1/lib-python/2.7/distutils/dist.py", line 972, in run_command
      cmd_obj.run()
    File "/opt/python/pypy-5.7.1/lib-python/2.7/distutils/command/build.py", line 127, in run
      self.run_command(cmd_name)
    File "/opt/python/pypy-5.7.1/lib-python/2.7/distutils/cmd.py", line 334, in run_command
      self.distribution.run_command(command)
    File "/opt/python/pypy-5.7.1/lib-python/2.7/distutils/dist.py", line 972, in run_command
      cmd_obj.run()
    File "/home/travis/gopath/src/github.com/asottile/dockerfile/.tox/pypy/site-packages/setuptools/command/build_ext.py", line 75, in run
      _build_ext.run(self)
    File "/opt/python/pypy-5.7.1/lib-python/2.7/distutils/command/build_ext.py", line 350, in run
      self.build_extensions()
    File "/opt/python/pypy-5.7.1/lib-python/2.7/distutils/command/build_ext.py", line 459, in build_extensions
      self.build_extension(ext)
    File "/home/travis/gopath/src/github.com/asottile/dockerfile/.tox/pypy/site-packages/setuptools_golang.py", line 111, in build_extension
      _check_call(cmd_build, cwd=pkg_path, env=env)
    File "/home/travis/gopath/src/github.com/asottile/dockerfile/.tox/pypy/site-packages/setuptools_golang.py", line 66, in _check_call
      subprocess.check_call(cmd, cwd=cwd, env=dict(os.environ, **env))
    File "/opt/python/pypy-5.7.1/lib-python/2.7/subprocess.py", line 186, in check_call
      raise CalledProcessError(retcode, cmd)
  CalledProcessError: Command '(u'go', u'build', u'-buildmode=c-shared', u'-o', '/tmp/pip-lRJy3v-build/build/lib.linux-x86_64-2.7/dockerfile.pypy-41.so')' returned non-zero exit status 2
  
  ----------------------------------------
  Failed building wheel for dockerfile
```

This stems from go treating `long` and `Py_ssize_t` as different types (despite `Py_ssize_t` being `typedef`d `long`)

I'm going to try and submit a patch that makes the function signatures actually `Py_ssize_t`.

--- a/.travis.yml
+++ b/.travis.yml
@@ -8,8 +8,8 @@ matrix:
             python: 3.5
         -   env: TOXENV=py36 GO=1.7
             python: 3.6
-        -   env: TOXENV=pypy GO=1.7
-            python: pypy-5.3.1
+        -   env: TOXENV=pypy GO=1.10
+            python: pypy
 install:
     - |
         export GOPATH=~/gopath
@@ -19,8 +19,6 @@ install:
         rsync -az . "$TRAVIS_BUILD_DIR"
         cd "$TRAVIS_BUILD_DIR"
         eval "$(gimme $GO)"
-    # workaround for #5
-    - pip install 'cryptography<2'
     - pip install coveralls tox
 script:
     - go get -t

